src/main/resources/templates/aboutus.html

```diff
@@ -17,7 +17,7 @@
 </div>
 <div>
     <h1>Moody Bluez - About Us</h1>
-    <p>
+    <p style="color: antiquewhite">
```

Comment:
The default color was not readable and was not a best practice. It is best practice to use contrasting colors on HTML views.
Citation - "[Accessibility Mozilla Developer](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)"

Reply:
Move this to external css... external css better than inline.